sdk/communication/arm-communication/README.md

```diff
@@ -63,8 +62,23 @@ const client = new CommunicationServiceManagementClient(new DefaultAzureCredenti
 // });
 // const client = new CommunicationServiceManagementClient(credential, subscriptionId);
 ```
+### Create Communication Service Resource with 'Managed Identity'
 
-
+```javascript
+// Existing resource creation code
+client.communicationServices.beginCreateOrUpdate(resourceGroup, resourceName, resource)
+
+// Create Resource with Managed Identity
+const resource = {
+    location: "Global",
+    dataLocation: "United States",
+    identity: {
+        type: "SystemAssigned" //or UserAssigned
+    }
+};
+// Defining an object that specifies identity type as System Assigned for the new resource.
+const result = await client.resources.createOrUpdate(resourceGroup, resourceName, resource);
```

Comment:
should be something like `client.communicationServices.beginCreateOrUpdate(resourceGroup, resourceName, resource);` here ?